net-voip/mumble/mumble-1.5.517.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="Low latency encrypted VoIP client"
+DESCRIPTION="Mumble is a low-latency, high quality voice chat program primarily \
+intended for gaming. \
+Features include a encrypted connection with Public/private-key authentication and \
+noise suppression. \
+To use Mumble, you need a mumble server. You can either join a public existing server\
+or host your own."
+HOMEPAGE="https://www.mumble.info/"
+COPYRIGHT="2005-2022 The Mumble Developers"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/mumble-voip/mumble/releases/download/v1.5.517/mumble-1.5.517.tar.gz"
+CHECKSUM_SHA256="8efd06a0f47ebfe27735f7575ba0becfb21d0f400a5082202dbbd998ab8351de"
+PATCHES="mumble-1.5.517.patchset"
+ADDITIONAL_FILES="mumble.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mumble$secondaryArchSuffix
+	app:Mumble
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix >= 1.83.0
+	lib:libexpat$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:libpcre2_posix$secondaryArchSuffix
+	poco$secondaryArchSuffix
```

Comment:
Replace with required lib:libraries, in this case it needs:
* libPocoFoundation
* libPocoXML
* libPocoZip

Reply:
poco package needs some changes for this to work, there is no $libVersion/$libVersionCompat for this ($libVersion = 94), so using lib:lib* won't work for this.

Reply:
I had it that way, but I could not install the package as it did not find poco. I'll have a look at the poco recipe

Reply:
Revbumped poco earlier today, you can now use lib:lib* and devel:lib* :)

Reply:
> I could not install the package as it did not find poco.

I had that earlier too, hence the revbump for poco.

Reply:
thank you